### DISCUSSION TITLE
More granular date timestamp on breaches

### DISCUSSION THREAD

**Discussion by bertold** [XposedOrNot/XposedOrNot-API#7] [Ideas]
Would it be possible to provide month and day in the breach-analytics endpoint for each breach?

**Comment by DevaOnBreaches** [comment]
Hello @bertold,

Thank you for your feedback and suggestion regarding the breach-analytics endpoint. I appreciate your interest in improving our API.

The breach-analytics endpoint currently provides year-based summaries for each breach, as it is designed to serve high-level details for presentation purposes. However, you've pointed out a valid use case for more granular information, and I understand that having the month and day components could be valuable for certain scenarios.

To accommodate this request, we can consider enhancing the endpoint to include the month for each breach. While I can't guarantee absolute date/day accuracy due to the nature of the data, we can certainly provide the month and year component to offer a more detailed insight.

In the meantime, if you require access to the year component, I would recommend utilizing the following API route:

```shell
https://api.xposedornot.com/v1/analytics/[email])
```

This route provides the year component and can complement the breach-analytics endpoint.

Please let me know if including the month and year in the breach-analytics endpoint aligns with your needs, or if you have any further suggestions or requirements. Your feedback is valuable in helping us improve our services.

Thank you for using our API, and I look forward to hearing from you.

**Comment by DevaOnBreaches** [comment]
Moving this to closure with 1e958da
